pkgs/tools/package-management/nix/default.nix

```diff
@@ -113,6 +113,13 @@ let
     hash = "sha256-s1ybRFCjQaSGj7LKu0Z5g7UiHqdJGeD+iPoQL0vaiS0=";
   };
 
+  patch-rapidcheck-shared = fetchpatch2 {
```

Comment:
FYI fetchpatch2 is only really required when renaming files ATM

Reply:
oh OK my bad. I just assumed it was just the "new and improved version" which should be used by default.

Reply:
I don't think it really matters, just that the normalisation is just slightly different and has a few bug fixes so not all hashes are compatible.